.gitlab/fips_compliance/fips_compliance_e2e.yml

```diff
@@ -0,0 +1,31 @@
+---
+.fips-compliance-e2e:
+  stage: check_fips_compliance
+  extends:
+    - .new_e2e_template
+  rules:
+    - !reference [.on_subcommands_or_e2e_changes]
+    - !reference [.manual]
+  variables:
+    TARGETS: ./tests/fips-compliance
+    TEAM: agent-shared-components
+
+fips-compliance-e2e-linux:
+  extends: .fips-compliance-e2e
+  needs:
+    - !reference [.needs_new_e2e_template]
+    - qa_agent
+  parallel:
+    matrix:
+      - EXTRA_PARAMS: --run "TestLinuxFIPSComplianceSuite"
+      - EXTRA_PARAMS: --run "TestFIPSCiphersSuite"
+
+
+fips-compliance-e2e-windows:
+  extends: .fips-compliance-e2e
+  needs:
+    - !reference [.needs_new_e2e_template]
+    - deploy_windows_testing-a7
+  parallel:
+    matrix:
+      - EXTRA_PARAMS: --run "TestWindowsFIPSComplianceSuite"
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏Can you add a newline?

Reply:
5f7018a3c8ed752b26c1a27ab728ee75dd27dee0